sql-cli/sql_cli/project.py

```diff
@@ -76,22 +82,34 @@ def airflow_config(self) -> dict[str, Any]:
         parser.read(filename)
         return {section: dict(parser.items(section)) for section in parser.sections()}
 
-    def update_config(self, environment: str = DEFAULT_ENVIRONMENT) -> None:
+    def _initialise_global_config(self) -> None:
         """
-        Sets custom Airflow configuration in case the user is not using the default values.
+        Initialises global config file that includes configuration to be shared across environments including the
+        airflow config.
+        """
+        config = Config(environment=GLOBAL_CONFIG, project_dir=self.directory)
+        global_env_filepath = config.get_global_config_filepath()
+        config.write_value_to_yaml("airflow", "home", str(self._airflow_home.resolve()), global_env_filepath)
+        if not Path.exists(self._airflow_dags_folder):
+            raise FileNotFoundError(f"Specified DAGs directory {self._airflow_dags_folder} does not exist.")
```

Comment:
The directory does not exist - not the file. Or is it common to raise this error also for directories? 🤔

Reply:
I did not find an exception for DirectoryNotFound . Upon reading docs and comments found that this is the exception that we could use and hence used it. https://docs.python.org/3/library/exceptions.html#FileNotFoundError

Reply:
Thanks for looking this up. Sounds good to me then. :)